packages/flutter/lib/src/gestures/drag.dart

```diff
@@ -182,7 +182,7 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
       _initialPosition = event.position;
       _pendingDragOffset = Offset.zero;
       if (onDown != null)
-        onDown(new DragDownDetails(globalPosition: _initialPosition));
+        invokeCallback('onDown', () => onDown(new DragDownDetails(globalPosition: _initialPosition)));
```

Comment:
We should also update these calls to give the expected return value (use `Null` where you expect `void`), as in:
```dart
  invokeCallback/*<Null>*/('onDown', () => onDown(new DragDownDetails(globalPosition: _initialPosition)));
```
...or for the later one:
```dart
  drag = invokeCallback/*<Drag>*/('onStart', () => onStart(initialPosition));
```